tools/should-skip-node

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -eu
+
+# Print '1' if the building of node/npm-dependent components should be skipped
+
+srcdir=`dirname $0`
+test -z "$srcdir" && srcdir=.
+top_srcdir="${srcdir}/.."
+
+(test -f "${top_srcdir}/src/ws/cockpit.service.in") || {
+    echo -n "**Error**: Directory "\`$srcdir\'" does not look like the"
+    echo " top-level Cockpit directory"
```

Comment:
That looks like overzealous line splitting.

Reply:
That's just copy/paste from `autogen.sh`